web/app/view/BasePermissionsController.js

```diff
@@ -36,7 +36,7 @@ Ext.define('Traccar.view.BasePermissionsController', {
                         var i, index;
                         if (success) {
                             for (i = 0; i < records.length; i++) {
-                                index = this.getView().getStore().find('id', records[i].getData().id);
+                                index = this.getView().getStore().findExact('id', records[i].getData().id);
```

Comment:
Frankly, do not now. I used old code:
https://github.com/tananaev/traccar/blob/v3.5/web/app/view/UserDevicesController.js#L37

But sure `getById` will be better